src/common/components/ui/formik/SingleNodeCheckbox.tsx

```diff
@@ -16,6 +18,9 @@ const SingleNodeCheckbox: React.FC<SingleNodeCheckboxProps> = ({
   idPostfix,
   ...props
 }) => {
+  const {
+    values: { openshiftVersion },
+  } = useFormikContext<ClusterCreateParams>();
```

Comment:
`values` is not being used anywhere in this file

Reply:
openshiftVersion is used, this is nested object deconstructing